docs/advanced/advanced.md

```diff
@@ -316,12 +317,34 @@ It is recommended to turn off firewalld:
 systemctl disable firewalld --now
 ```
 
+If you wish to keep firewalld enabled, the following rules are required:
```

Comment:
maybe note that these are the default pod and service CIDRs, and would need to be adjusted if the cidrs are changed?